libs/stack/stack-ui/src/components/fields/Select/Select.stories.jsx

```
@@ -1,42 +1,73 @@
   render: Template.bind({}),
   name: 'Disabled',
-
   args: {
+    name: 'season-disabled',
     disabled: true,
     icon: 'ChevronLeft',
+    options: defaultOptions,
   },
 }
 
 export const Error = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)